vowpalwabbit/cbify.cc

```diff
@@ -2,34 +2,56 @@
 // individual contributors. All rights reserved. Released under a BSD (revised)
 // license as described in the file LICENSE.
 
+#include <cfloat>
+#include <vector>
 #include "reductions.h"
 #include "cb_algs.h"
 #include "vw.h"
 #include "hash.h"
 #include "explore.h"
-
-#include <vector>
+#include "prob_dist_cont.h"
+#include "debug_log.h"
+#include <experimental/filesystem>
```

Comment:
This header is not available in C++11, must be removed

Reply:
Done.